parley/Cargo.toml

```diff
@@ -18,21 +18,37 @@ workspace = true
 
 [features]
 default = ["system"]
-std = ["fontique/std", "harfrust/std", "peniko/std", "skrifa/std", "swash/std"]
-libm = ["fontique/libm", "peniko/libm", "skrifa/libm", "swash/libm", "dep:core_maths"]
+std = ["fontique/std", "harfrust/std", "peniko/std", "skrifa/std"]
+libm = ["fontique/libm", "peniko/libm", "skrifa/libm", "dep:core_maths"]
 # Enables support for system font backends
 system = ["std", "fontique/system"]
 accesskit = ["dep:accesskit"]
 
 [dependencies]
-swash = { workspace = true }
 skrifa = { workspace = true }
 linebender_resource_handle = { workspace = true }
 fontique = { workspace = true }
 core_maths = { version = "0.1.1", optional = true }
 accesskit = { workspace = true, optional = true }
 hashbrown = { workspace = true }
 harfrust = { workspace = true }
+icu_collections = { workspace = true }
+# Used in ICU4X baked data sources, which depend on compiled data, but just locale fallback rules.
+icu_locale = { workspace = true }
+icu_normalizer = { workspace = true }
+icu_properties = { workspace = true, features = ["unicode_bidi"] }
+icu_provider = { workspace = true }
+icu_segmenter = { workspace = true, features = ["auto"] }
+unicode-bidi = { workspace = true }
+# Used in ICU4X baked data sources
+zerovec = { workspace = true }
```

Comment:
Is this actually used anywhere? Or are we specifying it here rather than as a transitive dependency deliberately?